development.ini

```diff
@@ -19,10 +19,11 @@ exports-allowable-types =
     pdf:pdf,application/pdf,PDF,PDF file, for viewing content offline and printing.
     epub:epub,application/epub+zip,EPUB,Electronic book format file, for viewing on mobile devices.
     zip:zip,application/zip,Offline ZIP,An offline HTML copy of the content.  Also includes XML, included media files, and other support files.
-##logging-configuration-filepath = <logging.ini (must be absolute path)>
+##
+#logging-configuration-filepath = /Users/richhart/git/cnx-archive/cnxarchive/default-logging.ini
 
 # OpenStax Accounts database connection info
-accounts.fdw.db-connection-string = dbname=oscaccounts-testing host=localhost port=5432 user=accounts password=accounts
+accounts.fdw.db-connection-string = dbname=accounts host=localhost port=5432 user=accounts password=openstaxaccounts
```

Comment:
I think it's good to change the `dbname` to accounts, but I'd prefer to keep `password` as accounts.  So now `dbname`, `user`, `password` are all accounts.